static/script-tests/tests/devices/device.js

```diff
@@ -112,7 +112,10 @@ require(
                     timeout: 1000
                 };
 
-                device.loadURL('http://test.uri/', opts);
+                device = new Device({'networking': { 'supportsCORS': true }});
+                spyOn(device, '_newXMLHttpRequest').and.returnValue(mockXMLHttpRequest);
```

Comment:
We shouldn't be mocking out private functions of the object under test, but I can see that this is how the test was working before (see line 26).